internal/rtsp/rtsp.go

```diff
@@ -237,7 +238,10 @@ func tcpHandler(conn *rtsp.Conn) {
 	})
 
 	if err := conn.Accept(); err != nil {
-		if err != io.EOF {
+		if err == rtsp.FailedAuth {
+			rAddr := conn.Connection.RemoteAddr
+			log.Warn().Msg(fmt.Sprintf("[rtsp] failed authentication from %s", rAddr))
```

Comment:
please use Msgf to avoid unnecessary string formatting and memory allocations if warn level is disabled.
And since the logger is structured, you can put the address to a separate field, so later the logs can be filtered by address field.
```suggestion
			log.Warn().Str("remote_addr", conn.Connection.RemoteAddr).Msg("[rtsp] failed authentication")
```

Reply:
Thanks! Fixed.